### DISCUSSION TITLE
Is it possible to include only output project in commopent rather than all the dlls and other projects files.

### DISCUSSION THREAD

**Discussion by HarshSuman82** [wixtoolset/issues#6492] [Questions]
![image](https://user-images.githubusercontent.com/86040662/123417484-71448b80-d5d5-11eb-8206-883e2a73081a.png)

**Comment by BMurri** [comment]
You can harvest a Visual Studio project using the Harvest tool (aka `Heat`).

If you do, I'd strongly suggest using the tool just once and then maintain the generated WXS file. If you determine you must harvest in your build, I'd very strongly recommend using the `-ag` flag (which sets the guid of each component to "*"), but if you have already released your product with the guids from your image, then please don't harvest in your build (you REALLY do not want to be changing your component guids from build to build unless you are changing installation directories).
